ksqldb-engine/src/test/java/io/confluent/ksql/structured/SchemaKSourceFactoryTest.java

```diff
@@ -136,7 +136,6 @@ public void shouldBuildWindowedStream() {
   public void shouldBuildNonWindowedStream() {
     // Given:
     when(dataSource.getDataSourceType()).thenReturn(DataSourceType.KSTREAM);
-    when(keyFormat.isWindowed()).thenReturn(false);
```

Comment:
It seems I wasn't clear with the scope of my previous comment: I think it's fine to leave this mock in here even though mockito would default to false anyway, but we should remove the feature flag default since feature flag defaults are assumed to be false unless otherwise specified, while there's no default assumption for windowed vs unwindowed for sources.